proposals/NNNN-new-remote-control-modules-and-parameters.md

```diff
@@ -0,0 +1,512 @@
+# New remote control modules (LIGHT, ADUIO, HMI_SETTINGS) and parameters (SIS Data)
+
+* Proposal: [SDL-NNNN](NNNN-new-remote-control-modules-and-parameters.md)
+* Author: [Zhimin Yang](https://github.com/yang1070)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Android / RPC ]
+
+## Introduction
+
+SDL remote control provides the ability for mobile applications to control certain components of the vehicle, such as, tune the radio, set the desired temperature or change the fan speed. However, currently only `CLIMATE` and `RADIO` modules are available for remote-control. Mobile application developers need more than basic climate or radio control. To meet developers' requirements, we propose to add three new modules (`AUDIO`, `LIGHT`, `HMI_SETTINGS`) and some new parameters (heatedXXXEnable and radio sisData) to the existing `CLIMMATE` and `RADIO` modules respectively. This proposal only defines new data types and new parameters added to the existing data types. It does not add or change any remote control RPCs.
```

Comment:
Please fix `CLIMATE` here and in the PR description.

Reply:
thanks, updated CLIMMATE to CLIMATE